client/me/purchases/cancel-purchase/domain-options.jsx

```diff
@@ -116,6 +121,40 @@ const CancelPurchaseDomainOptions = ( {
 		</div>
 	);
 
+	const CancelableDomainTransferMessage = () => (
+		<div>
+			<p>
+				{ translate(
+					'It seems you have a pending domain transfer for the domain %(mappedDomain)s. ' +
+						'In order to receive a full refund, please cancel the domain transfer before requesting the cancellation of your plan.',
+					{
+						args: {
+							mappedDomain: includedDomainTransfer.meta,
+						},
+					}
+				) }
+			</p>
+			<p>
+				{ translate(
+					'You will receive a partial refund of %(refundAmount)s which is %(planCost)s for the plan ' +
+						'minus %(domainCost)s for the domain.',
+					{
+						args: {
+							domainCost: includedDomainTransfer.priceText,
+							planCost: planCostText,
+							refundAmount: purchase.refundText,
+						},
+					}
+				) }
+			</p>
+			<p>
+				{ translate(
+					'In some cases, a domain transfer has progressed too far to be canceled. Please contact support if you have questions about this'
```

Comment:
As before, I think this sentence should end in a period.

Reply:
This sentence was removed as per [David’s comment](https://linear.app/a8c/issue/CHE-6/improve-the-messaging-in-the-refund-page#comment-53249bd1). Happy to re-add it (with the right punctuation)